docs/quick-start/ignite-commands.md

```diff
@@ -0,0 +1,138 @@
+# Ignite Commands
+
+This document reviews the commands that the Ignite 2.0 CLI provides. This guide
+assumes that you've installed Ignite. If you haven't, then check out our Getting
+Started Guide [here]().
+
+Enter `ignite` into your command line to see the commands offered by Ignite 2.0.
+
+
+### Add
+
+```
+ignite add {ignite plugin name}
+```
+
+Adds an Ignite plugin.
+
+Ignite plugins are just node plugins. They must have an index.js file with an
+`add` function. Calling `ignite add <plugin name>` calls this function. Its job is to add the
+desired plugin to your project. It may modify files in your project.
```

Comment:
It will modify files.  Almost always.  Typically they add other dependencies or example files.  `ignite-*` plugins by themselves are `devDependencies`.  Ignite plugins by themselves do not go into your product.